specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -1614,6 +1614,208 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
+      "description": "A managed http route to update",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      }
+    },
+    "ManagedHttpRoute": {
+      "description": "Managed certificates used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Certificate resource specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "The scale configuration of the session pool.",
+              "$ref": "#/definitions/HttpRouteProvisioningState"
+            },
+            "fqdn": {
+              "description": "FQDN of the route resource.",
+              "type": "string",
+              "readOnly": true
+            },
+            "customDomains": {
+              "description": "custom domain bindings for http Routes' hostnames.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/CustomDomain"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "rules": {
+              "description": "Routing Rules for http route resource.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HttpRouteRule"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            }
+          }
+        }
+      },
+      "x-ms-client-flatten": true
+    },
+    "CustomDomain": {
+      "description": "Custom Domain of a Container App",
+      "type": "object",
+      "required": [
+        "name"
+      ],
```

Comment:
Isn't this an existing type? Do we need to define it all over again?

Reply:
you are right. let me see if i can add the reference

Reply:
lets keep this here